pkg/network/http/compile.go

```diff
@@ -22,6 +23,13 @@ func getRuntimeCompiledHTTP(config *config.Config) (runtime.CompiledOutput, erro
 
 func getCFlags(config *config.Config) []string {
 	var cflags []string
+
+	if rt.GOARCH == "arm64" {
```

Comment:
In the current state of `main`, I don't think this is necessary since we reverted the PR that needed it.

Reply:
Fair enough, I'll remove this for now & we can add it back along with the relevant change in the future